spaces/domains.json

```
@@ -260,5 +260,6 @@
   "vote.planetfinance.io": "planetfinance.eth",
   "vote.frenchie.so": "frenchienetwork.eth",
   "gov.50cent.network": "50centnetwork.eth",
-  "vote.kogecoin.io": "kogecoin.eth"
+  "vote.kogecoin.io": "kogecoin.eth",
+  "vote.tea-swap.finance" : "tea-swap.eth"
```

Please remove this space.
```suggestion
  "vote.tea-swap.finance": "tea-swap.eth"
```